presto-jdbc/src/main/java/io/prestosql/jdbc/PrestoDatabaseMetaData.java

```diff
@@ -1353,8 +1361,27 @@ public boolean autoCommitFailureClosesAllResultSets()
     public ResultSet getClientInfoProperties()
```

Comment:
Is semantic of this method is well defined or is it implemention just depends on vendor?